Backend/src/v1/testing.py

```diff
@@ -0,0 +1,131 @@
+# own
+from datetime import time
+import os
+import random
+from ..types import TestingResponse, DatasetsResponse
+from ..utils import (
+    download_qaqc_source_dataset as util_download_qaqc,
+    get_qaqc_info_dict as util_get_info_dict,
+)
+from .dataset import upload_dataset, get_all_processed_datasets
+from .data import get_all_labels, get_all_nodes
+from .run_ml import run, get_status
+
+#pip
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.testclient import TestClient
+
+import asyncio
+import time
+import io
+import json
+
+
+
+router = APIRouter(
+    prefix="/testing",
+    tags=["Testing"],
+)
+
+@router.post("/")
+async def manual_test_sequence(

```

Comment:
I think it would be nice to have the option to change the model instead of always using `qaqc_main`. (Maybe set `qaqc_main` as a default but with the possibility to overwrite?)

I also personally think it would be nice to have a default value in the `dim_red_method`, just making it a little faster to run tests.

Reply:
done